apps/etterlatte-saksbehandling-ui/client/src/shared/header/Enhet.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components'
+import { useAppSelector } from '~store/Store'
+import { Select } from '@navikt/ds-react'
+import React, { useEffect, useState } from 'react'
+import { IEnhet } from '~store/reducers/SaksbehandlerReducer'
+
+const isNotValid = (value: string | null): value is string => [null, undefined, ''].includes(value)
+
+export const Enhet = () => {
+  const [valgtEnhet, setValgtEnhet] = useState<IEnhet | undefined>()
+  const { enheter } = useAppSelector((state) => state.saksbehandlerReducer.saksbehandler)
+
+  useEffect(() => {
+    if (!enheter.length) {
+      localStorage.removeItem('enhet')
+      return
+    }
+
+    const enhet = localStorage.getItem('enhet')
```

Comment:
Hvorfor legger vi denne infomarsjonen på localstorage mens saksbehandlerident legges kun i redux? Har vi behov for å ha det der?

Reply:
Lagring i redux innebærer vel at verdien må hentes ved hver eneste oppdatering av siden? Så vidt jeg vet har vi ikke implementert state hydration, e.l.

Reply:
Jepp. Samtidig burde de vel behandles likt, så noe sesjonsdata med enhet og saksbehandler etter at bruker har valgt denne enhet feks. Blir en litt ekstra jobb da vi må legge det rundt alle kall.